.github/workflows/build-package-docs.yaml

```diff
@@ -140,8 +146,10 @@ jobs:
              -d '{"msgtype": "m.text", "body": "${{ env.FAIL_MESSAGE }}"}'
 
   deploy-package-docs:
```

Comment:
This could also have a needs on `check-deploy`, and then it won't run if that is skipped (see the comment above about adding an `if` to that job in turn to skip if docs build fails).

Reply:
I believe this already exists? https://github.com/ansible/ansible-documentation/pull/1814/files#diff-556c4624ec829c57c68bca98085f1fac2a96f6af1ff4dc3c42e4a8f20e2a10abR151

Reply:
Ah, doh, it does. I did not read the diff too closely, apparently.